app/controllers/cases/offender_sar_controller.rb

```diff
@@ -46,7 +46,9 @@ def create
       @case.current_step = params[:current_step]
       get_rejected
       load_optional_flags_from_params
-
+      if @rejected == "true"
+        set_deadlines
```

Comment:
This method is already called on creation - https://github.com/ministryofjustice/correspondence_tool_staff/blob/ea807f31e2a372b15838eec958c8ac127fcf6633/app/models/case/base.rb#L416

Reply:
amended